rollup.config.test.js

```diff
@@ -0,0 +1,21 @@
+import resolve from '@rollup/plugin-node-resolve';
```

Comment:
Why do we need to use rollup for the tests?

Reply:
Our source code uses node-isms for dependency structure (not pure ESM): 

```
submodule/
    |-- index.js
```

and importing like:

```javascript
import { a, b } from './submodule';
```

to remove this dependency, we would need to change these imports to:

```javascript
import { a, b } from './submodule/index';
```

Reply:
Oh sorry, I thought this was an issue with `@rollup/plugin-node-resolve` not `rollup` at all.

Our previous testing approach required bundling our source code for tests. This is still required because 1.) our source isn't actually ESM (so we can't run node directly), and 2.) we need to pipe CJS in browserify for `tape-run` for the electron tests.

Reply:
FWIW vite uses rollup & rollup-plugins under the hood, it's just I can't pipe the output to std out like we have previously.